### ISSUE TITLE
Latest grimoirelab pip package is uninstallable

### ISSUE THREAD

**Issue by jgbarah** [chaoss/grimoirelab#450]
It seems latest [grimoirelab package in pypi](https://pypi.org/project/grimoirelab/0.2.62/) (0.2.62) depends on 'sortinghat==0.7.16':

```
pip install grimoirelab==0.2.62
...
ERROR: Could not find a version that satisfies the requirement sortinghat==0.7.16 (from grimoirelab)
ERROR: No matching distribution found for sortinghat==0.7.16
```

However, the latest [version of sortinghat I see in pypi](https://pypi.org/project/sortinghat/#history) is 0.7.15.

Curiously enough, the latest commit in the master branch of sortinghat is "Update version number to 0.7.16". Maybe this version was released, but never uploaded to pypi? If so, please upload it there, and I can check if the latest `grimoirelab` package installs.

**Comment by jgbarah** [comment]
I'm opening [#587 in grimoirelab-sortinghat](https://github.com/chaoss/grimoirelab-sortinghat/issues/587) asking for the upload of sortinghat 0.7.16 to pypi.

**Comment by jgbarah** [comment]
BTW, I think we could close #315, which is (I think) irrelevant now, since versions are newer.

**Comment by jgbarah** [comment]
Thanks for uploading sortinghat 0.7.16. But something wrong is happening. Now, when I try to install, there is a dependency of grimoirelab 0.2.62 on grimoirelab 0.2.61 (?):

```
pip install grimoirelab==0.2.62
....
ERROR: Cannot install grimoirelab==0.2.62 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested grimoirelab==0.2.62
    grimoirelab 0.2.62 depends on grimoirelab==0.2.61

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

Could you have a look at that, please?

**Comment by vchrombie** [comment]
Hi @jgbarah 
The GrimoireLab version 0.2.63 has been released recently and it looks the release commit changes look good.

This should solve the issue you are facing.

**Comment by zhquan** [comment]
The issue is fixed in version `0.2.64` @jgbarah @vchrombie

**Comment by jgbarah** [comment]
Yes, i checked it. Thanks a lot! I'm closing the issue.
